### PR TITLE
haa binary downloader

### PR DESCRIPTION
**Pull request by wayn3**
Hey RS,

would you be interested in including this binary haa_downloader.sh bash script in the haa repository?

### CODE REVIEW

**Comment by RavenSystem** [comment]
You must include a LICENSE header, or terms of use... into your file.

File: haa_downloader.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        if [ -z "$validation" ]
+        then
+                if [ ! -d "$version" ]; then
+                        mkdir "$version"
+                fi
+                for i in "${files[@]}"
+                do
+                        file_name="$(basename $i)"
+                        echo -n "Saving ./$version/$file_name ... "
+                        wget -qO "$version/$file_name" "$i"
```
Comment (RavenSystem) [inline haa_downloader.sh]: Better use curl instead wget

Comment (wayn3) [reply haa_downloader.sh]: Started with just wget calls to the latest links before I decided to add in other stuff, I’ll update it to use curl commands and include something in the header

Comment (wayn3) [reply haa_downloader.sh]: replaced wget with curl,
reduced github api calls to 2 and included a count
included a gpl license statement in the code and printed to terminal

**Pull request merged**